app/Database/migrations/2024_01_26_142501_add_vatsim_id_to_users.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('vatsim_id')->default('')->after('discord_private_channel_id');
```

Comment:
Not sure about this...

Back in time, when asked @nabeelio kindly replied us to NOT have IVAO/VATSIM stuff in the main users model/table, instead wanted us to use the custom profile fields for this kind of external stuff.

So, if we are adding VATSIM ID to users model directly, then we should consider adding IVAO ID (and maybe others like PilotEdge and PosCon).

Reply:
The `vatsim_id` is there because that's how we set up the OAuth. We discussed the `discord_id`, and we decided to keep it there, in the way OAuth is impletemented, it's more efficient to have it directly in the same table. Since the OAuth implementation I've done is something generic, not provider-dependent, if we move the `vatsim_id` elsewhere, we also need to move the `discord_id` or change the way oauth is implemented. 
Other networks aren't required yet because we don't use OAuth with them. What makes it required here is relaly the OAuth implementation

Reply:
For me, he approved having the `provider_id` from OAuth in the users table during the OAuth implementation. So, if our provider is vatsim, naturally, we add `vatsim_id`